hazelcast/src/test/java/com/hazelcast/test/mocknetwork/TestNodeRegistry.java

```diff
@@ -32,9 +33,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 
-import static org.junit.Assert.assertEquals;
-
-public final class TestNodeRegistry {
+public final class TestNodeRegistry extends HazelcastTestSupport {
```

Comment:
Ah, I see. I saw this construct in some other places, so I thought I might copy it. But this is better indeed.